Resources/Locale/en-US/bloodstream/bloodstream.ftl

```diff
@@ -0,0 +1 @@
+﻿bloodstream-component-looks-pale = [color=bisque]{CAPITALIZE(SUBJECT($target))} looks pale.[/color]
```

Comment:
this is incorrect for epicene, `They looks pale` but going to fix in a separate PR